retrieve.go

```diff
@@ -6,28 +6,27 @@ import (
 )
 
 // Get retrieves the value at path of struct(s).
-func Get(out interface{}, path string) interface{} {
-	result := get(reflect.ValueOf(out), path)
-
-	if result.Kind() != reflect.Invalid && !result.IsZero() && result.CanInterface() {
-		return result.Interface()
-	}
-
-	return nil
-}
-
-func GetAllowZero(out interface{}, path string) interface{} {
+func Get(out interface{}, path string, allowZero bool) interface{} {
```

Comment:
use a [functional option](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis) to avoid breaking change.

Reply:
Didn't understand you first comment fully, sorry. Fixed with functional options,  I added a global Get function. Wasn't sure if to make the Receiver exported or not, currently unexported, what do you think?

Reply:
@thoas is this version ok?